orca-core-tck/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepositoryTck.groovy

```diff
@@ -628,14 +626,14 @@ abstract class ExecutionRepositoryTck<T extends ExecutionRepository> extends Spe
 
     where:
     statuses             | limit | type          | expectedResults
-    []                   | 0     | PIPELINE      | ExecutionStatus.values().size()
-    []                   | 0     | ORCHESTRATION | ExecutionStatus.values().size()
-    [RUNNING, SUCCEEDED] | 0     | PIPELINE      | 2
-    [RUNNING, SUCCEEDED] | 0     | ORCHESTRATION | 2
-    []                   | 1     | PIPELINE      | 1
-    []                   | 1     | ORCHESTRATION | 1
-    [RUNNING, SUCCEEDED] | 1     | PIPELINE      | 1
-    [RUNNING, SUCCEEDED] | 1     | ORCHESTRATION | 1
+    []                   | 0     | Execution.ExecutionType.PIPELINE      | ExecutionStatus.values().size()
+    []                   | 0     | Execution.ExecutionType.ORCHESTRATION | ExecutionStatus.values().size()
+    [RUNNING, SUCCEEDED] | 0     | Execution.ExecutionType.PIPELINE      | 2
+    [RUNNING, SUCCEEDED] | 0     | Execution.ExecutionType.ORCHESTRATION | 2
+    []                   | 1     | Execution.ExecutionType.PIPELINE      | 1
+    []                   | 1     | Execution.ExecutionType.ORCHESTRATION | 1
+    [RUNNING, SUCCEEDED] | 1     | Execution.ExecutionType.PIPELINE      | 1
+    [RUNNING, SUCCEEDED] | 1     | Execution.ExecutionType.ORCHESTRATION | 1
```

Comment:
It would be nice if all these refactored import statements got rolled back. It's still a hassle to review the PR as-is.

Reply:
👍 will do